pyproject.toml

```diff
@@ -134,6 +136,10 @@ fixable = ["ALL"]
 exclude = [
     "cpp/scripts/gdb-pretty-printers.py",
 ]
+extend-unsafe-fixes = [
+    # zip-without-explicit-strict's autofix sets strict to False, but we want to enforce True
```

Comment:
An inexact check could be to add a `pygrep` rule for `strict=False` in `.pre-commit-config.yaml`. Not sure how bad the false positives would be.